src/MQTTProtocolOut.c

```diff
@@ -341,6 +341,7 @@ int MQTTProtocol_handlePingresps(void* pack, int sock)
 	client = (Clients*)(ListFindItem(bstate->clients, &sock, clientSocketCompare)->content);
 	Log(LOG_PROTOCOL, 21, NULL, sock, client->clientID);
 	client->ping_outstanding = 0;
+	free(pack);
```

Comment:
According to a comment on MQTTAsyncUtils.c:3028, handle functions are supposed to free the pack memory when finished. All the others do, this one did not.